R/DataPulls.R

```diff
@@ -591,4 +598,170 @@ getCohortMethodResult <- function(connection = NULL,
     dplyr::select(-definition)
   
   return(result)
-}
\ No newline at end of file
+}
+
+
+
+#' Get Power data
+#'
+#' @details
+#' Get the power results for target, comparator and outcome cohorts in data frame format from
+#' OHDSI comparative effectiveness data model. The output maybe used to create plots or tables.
+#'
+#' @template connection
+#' @template databaseSchema
+#' @template targetId
+#' @template comparatorId
+#' @template databaseId
+#' @template analysisId
+#' @template outcomeId
+#' @param cohortMethodResult   (optional) A R-dataFrame object with fields named in camelCase containing
+#'                             data from cohort_method_result table in comparative effectiveness data model.
+#' @param cohortMethodAnalysis (optional) A R-dataFrame object with fields named in camelCase containing
+#'                             data from cohort_method_analysis table in comparative effectiveness data model.
+#' @return                     Tibble. 
+#'
+#' @examples
+#' \dontrun{
+#' connectionDetails <- createConnectionDetails(dbms = "postgresql",
+#'                                            user = "joe",
+#'                                            password = "secret",
+#'                                            server = "myserver")
+#' example1 <- getPower(connection = connectionDetails,
```

Comment:
Please add comment indicating the following statement is an example when using data in a database